packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -247,6 +250,7 @@ export interface DropdownState {
   value: ShorthandCollection<DropdownItemProps>;
   itemIsFromKeyboard: boolean;
   isFromKeyboard: boolean;
+  inverted: boolean;
```

Comment:
Why do we need `inverted` as state?

Reply:
That's the original bug that I was fixing, adding inverted state. I think it's a requirement for running in Teams?